### DISCUSSION TITLE
一分钟读论文：《30年软件重构研究：系统性文献综述》

### DISCUSSION THREAD

**Discussion by giscus** [unbug/unbug.github.io#9] [Announcements]
# 一分钟读论文：《30年软件重构研究：系统性文献综述》

如何评估重构的收益？IEEE 软件工程期刊登的论文《30 Years of Software Refactoring Research: A Systematic Literature Review》分析了过去30年3183篇关于重构的研究论文，对现有重构研究的最具可扩展性和最全面的文献综述。

https://unbug.github.io/30-Years-of-Software-Refactoring-Research-A-Systematic-Literature-Review/

**Comment by unbug** [comment]
重构前不知道如何系统性评估收益可以看看 IEEE 的这篇论文
